app/controllers/decidim/direct_verifications/verification/admin/imports_controller.rb

```diff
@@ -30,6 +31,23 @@ def create
 
             redirect_to new_import_path
           end
+
+          def workflows
+            workflows = configured_workflows & current_organization.available_authorizations.map.to_a
+            workflows.map do |workflow|
+              [t("#{workflow}.name", scope: "decidim.authorization_handlers"), workflow]
+            end
+          end
+
+          def configured_workflows
+            return Decidim::DirectVerifications.config.manage_workflows if Decidim::DirectVerifications.config
+
+            ["direct_verifications"]
+          end
+
+          def current_authorization_handler
+            params[:authorization_handler]
+          end
```

Comment:
These three methods are copied from the direct_verifications_controller and need a DRY. However, I think this falls out of the scope of this PR. It's not a big deal because it's very easy to see they are duplicates but it needs addressing. Do you agree on creating an issue @microstudi ?

For instance, I see that `#configured_workflows` would be unnecessary if `DirectVerifications.config.manage_workflows` returned `["direct_verifications"]` by default.

Reply:
yes let's not worry about this now. This is a big improvement already!